src/lib/ketrew_engine.ml

```diff
@@ -725,7 +718,7 @@ module Run_automaton = struct
       return (make_new_target ~log:("Attempt to start") starting_attemp)
     | `Eval_condition (condition, make_new_target) ->
       begin
-        Target.Condition.eval condition
+        Ketrew_eval_condition.bool condition
```

Comment:
I changed from `Ketrew_condition` to `Ketrew_eval_condition` to stress that `Condition.t` is not defined there (it's in the `pure` lib, while `eval` means “has to be within Unix”).